docs/integrate/etl/influxdb.md

```diff
@@ -0,0 +1,191 @@
+(influxdb-import)=
+# Import data from InfluxDB
+
+In this quick tutorial we use [influxio](https://github.com/daq-tools/influxio)
+to import data from InfluxDB into CrateDB. It can be used both as a standalone
+program, and as a library.
+
+:::{note}
+Please note that influxio is alpha-quality software, and a work in progress.
+Contributions of all kinds are very welcome, in order to make it more solid. 
+Breaking changes should be expected until a 1.0 release, so version pinning is
+recommended, especially when you use it as a library.
+:::
+
+(setup-influxio)=
+## Setup
+
+`influxio` can be installed with pip:
+
+:::{code} console
+pip install influxio
+:::
+
+You will also need the [cratedb-toolkit](https://github.com/crate-workbench/cratedb-toolkit/tree/main/cratedb_toolkit/io#installation)
+to load the data into your CrateDB instance. It can be installed with:
+
+:::{code} console
+pip install --upgrade 'cratedb-toolkit[all]'
+:::
```

Comment:
When aiming to use CrateDB Toolkit, installing `influxio` separately is not needed. The right installation command would be `pip install 'cratedb-toolkit[influxdb]'`.

In general, in the main section of the article, I would omit all details about influxio, and only present the corresponding CLI command of CrateDB Toolkit.

At the end of the article, I would link to the documentation of influxio, conveying the message that it offers a few more commands which help in different scenarios like the one you've picked.

Reply:
Currently, there is no `pip install` command inside. When aiming for conveying that tutorial "the Docker way", like proposed at https://github.com/crate/cratedb-guide/pull/75#issuecomment-2099100756, we will probably not need any of those. JFYI.